Dockerfile

```diff
@@ -87,7 +87,7 @@ RUN apt-get update && apt-get install -y \
 #            will need updating, to avoid errors. Ping #docker-maintainers on IRC
 #            with a heads-up.
 # IMPORTANT: When updating this please note that stdlib archive/tar pkg is vendored
```

Comment:
remove these `archive/tar` comment lines as well

Reply:
Oh! forgot about that one